handlers.go

```diff
@@ -96,11 +96,11 @@ func newFalcoPayload(payload io.Reader) (types.FalcoPayload, error) {
 
 	var kn, kp string
 	for i, j := range falcopayload.OutputFields {
-		if i == "k8s_ns_name" {
-			kn = j.(string)
+		if i == "k8s.ns.name" {
+			kn, _ = j.(string)
```

Comment:
I don't understand that change, it's equivalent with what was before

Reply:
The `, _` part will prevent it from causing a panic like so:

```
t, ok := i.(T)
If i holds a T, then t will be the underlying value and ok will be true.

If not, ok will be false and t will be the zero value of type T, and no panic occurs.
```
We don’t actually care about the value of `ok` so we just use `_`

`kn` will just be an empty string if `j` is `nil` or not castable to a string

Reply:
Ok got it. Thanks.